File: javaimagedialog/app/src/main/java/com/example/javaimagedialog/MainActivity.java
package com.example.javaimagedialog;

import android.app.Activity;
import android.app.Dialog;
import android.os.Bundle;
import android.widget.ImageView;
import android.widget.TextView;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Dialog dialog = new Dialog(this);
        dialog.setContentView(R.layout.custom_dialog);
        dialog.setTitle("Custom Dialog");

        TextView tv = (TextView) dialog.findViewById(R.id.text);
        tv.setText("Hello. This is a Custom Dialog !");

        ImageView iv = (ImageView) dialog.findViewById(R.id.image);
        iv.setImageResource(R.drawable.icon);

        dialog.show();
    }
}

File: spinner2/app/src/main/java/com/example/spinner/MainActivity.kt
package com.example.spinner

import android.R
import android.app.Activity
import android.app.AlertDialog
import android.app.Dialog
import android.os.Bundle
import android.widget.ImageView
import android.widget.TextView


class MainActivity : Activity() {

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var dialog = AlertDialog.Builder(this@MainActivity)
        dialog.setView(R.layout.custom_dialog)
        dialog.setTitle("Custom Dialog")
        val tv = dialog.findViewById(R.id.text) as TextView
        tv.text = "Hello. This is a Custom Dialog !"
        val iv: ImageView = dialog.findViewById(R.id.image) as ImageView
        iv.setImageResource(R.drawable.weight)
        dialog.show()
    }
}